main.go

```diff
@@ -282,7 +279,8 @@ func convertField(curPkg *ProtoPackage, desc *descriptor.FieldDescriptorProto, m
 
 func convertMessageType(curPkg *ProtoPackage, msg *descriptor.DescriptorProto, opts *protos.BigQueryMessageOptions) (schema []*Field, err error) {
 	if glog.V(4) {
-		glog.Info("Converting message: ", proto.MarshalTextString(msg))
+		b, _ := prototext.Marshal(msg)
+		glog.Info("Converting message: ", string(b))
```

Comment:
`glog.Info` will take care of calling the `String` receiver method, which will be a `prototext.Marshal` call with the best intended marshal options for debugging:
```suggestion
		glog.Info("Converting message: ", msg))
```
(I guess they didn’t trust that `github.com/golang/protobuf` messages implemented `fmt.Stringer`, but `google.golang.org/protobuf/proto` messages are guaranteed to do so.)

Reply:
Done